.babelrc

```diff
@@ -10,5 +10,13 @@
   ],
   "ignore": [
     "src/assets"
-  ]
+  ],
+  "overrides": [{
+    "test": "./src/helpers/import-helper.js",
```

Comment:
Don't think this one is needed right?

Reply:
My apologies, I should have rephrased it to be more specific. I'm not very knowledgeable in Babel, so I was wondering what this line would do and was confused with something else. But your explanation is very thorough, thanks for that!